index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio: Setup and Mobile First Web App</title>
+    <link rel="stylesheet" href="css/styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+    
+    
+   
+    <div class="container">   
+        
+        <div class="ellipse-11">
+            <img src="./images/Ellipse11.png" alt="Ellipse 11">
+        </div>
+        <img src="./images/Ellipse5.png" alt="Ellipse 5" class="ellipse-5">
+        <img src="./images/Rectangle7.png" alt="Rectangle 7" class="rectangle-7">
+        <img src="/images/Rectangle4.png" alt="Rectangle 4" class="rectangle-4">
+        
+        <div class="ellipse-3">
+            <img src="./images/Ellipse3.png" alt="ellipse 3">
+        </div>
+        <img src="./images/Ellipse12.png" alt="ellipse 12" class="ellipse-12">
+        <img src="./images/Ellipse13.png" alt="Ellipse 13" class="ellipse-13">
+        <img src="./images/Ellipse4.png" alt="Ellipse 4" class="ellipse-4">
+        <img src="./images/Rectangle5.png" alt="Rectangle 5" class="rectangle-5">
+        <img src="./images/Rectangle6.png" alt="Rectangle 6" class="rectangle-6">
+        <img src="./images/Ellipse8.png" alt="Ellipse 8" class="ellipse-8">
```

Comment:
- [ ] I kindly suggest that you make use of CSS background property for this images because they are purely for decoration purposes. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info.